ledger-core/src/CMakeLists.txt

```diff
@@ -163,7 +166,7 @@ target_link_libraries(ledger-core-interface INTERFACE ethash)
 target_link_libraries(ledger-core-interface INTERFACE ${SQLITE_LIB})
 
 target_include_directories(
-    ledger-core-interface INTERFACE
+    ledger-core-interface SYSTEM INTERFACE
```

Comment:
Why do you need `SYSTEM` here?

Reply:
The `SYSTEM` keyword is needed to avoid to propagate compiler warnings to our dependencies.
`cmake` will interpret `SYSTEM` by adding `-Isystem` to the compiler command line which will skip all headers after that option.